webrtc.html

```diff
@@ -3043,6 +3035,39 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
           </dd>
 
         </dl>
+        <dl class="idl" title="dictionary RTCRtpTransceiverInit">
```

Comment:
I moved it closer to where it is being used (in `addTransceiver`); at the moment, it is with `RTCRtpTransceiver`, but nothing in that interface makes use of `RTCRtpTransceiverInit`, with the result that the reader is left puzzled as to what it is used for.